products/pdt/pdt/src/main.rs

```diff
@@ -69,6 +69,8 @@ enum Commands {
     ParseEvents,
     #[command(name = "listen")]
     Listen(ListenOptions),
+    #[command(name = "test")]
+    Test,
```

Comment:
Remove?

Reply:
Keeping it around for now as `docker-compose` would need it

Reply:
Why does `docker-compose` need a command which prints `"Hello World"` then loops forever?

Reply:
I have currently added for testing purposes. I needed a way to keep the container running to inspect the volumes are connected properly. I will remove this command once all setup is completed. For now I am using it to investigate the inner workings of the pod. If you think there is an alternative way I could achieve this let me know

Reply:
Currently, I don't trust `listen` command to work properly yet as I have not managed to connect to the BQ. Wanted to break this down into separate steps. I am happy to add a `TODO:` comment to make sure I don't forget to remove it later